### PR TITLE
Update references

### PR DESCRIPTION
**Pull request by himkt**
Added the reference to jnlp.

### CODE REVIEW

**Comment by himkt** [comment]
@jun-harashima Please take a look. :pray:

File: README.md [hunk 1]
```diff
@@ -110,10 +110,19 @@ If you use the CPC for your research, please cite the following paper:
...
+  author    = "Jun Harashima and Makoto Hiramatsu",
+  title     = "{Cookpad Parsed Corpus: Linguistic Annotations of Japanese Recipes}",
+  booktitle = "Proceedings of the 14th Linguistic Annotation Workshop (LAW 2020)",
+  pages     = "87-92",
+  year      = "2020",
+}
+```
+
+```
+@improceedings{Hiramatsu2021,
```
Comment (jun-harashima) [inline README.md:122]: `inproceedings`

File: README.md [hunk 2]
```diff
@@ -110,10 +110,19 @@ If you use the CPC for your research, please cite the following paper:
...
+  author    = "平松 淳 and 原島 純",
+  title     = "レシピ解析の現状と課題: Cookpad Parsed Corpus を例として",
+  booktitle = "言語処理学会年次大会",
```
Comment (jun-harashima) [inline README.md:125]: - `言語処理学会年次大会` is the name of the conference. The booktitle is `言語処理学会第27回年次大会発表論文集`. I often add the abbreviation of the name of the conference (i.e., `NLP 2021`).
- You can add `pages  = 345-349` although it's an optional information.

**Comment by jun-harashima** [comment]
> This repository consists of a set of scripts that were used in the following paper:
> Harashima and Hiramatsu, Cookpad Parsed Corpus: Linguistic Annotations of Japanese Recipes, LAW, 2020.

Let's update this as well.

File: README.md (hunk 2 above)
Comment (himkt) [reply README.md]: 46720a2 and 2870194 updated the PR based on suggestions. :pray:

File: README.md (hunk 1 above)
Comment (himkt) [reply README.md]: fixed in 46720a2

**Comment by himkt** [comment]
So sorry @jun-harashima for the inactive. I revised my PR based on your suggestions.

**Review by jun-harashima — APPROVED** [review]

**Comment by jun-harashima** [comment]
LGTM 🚀

**Pull request merged**